cmd/client.go

```diff
@@ -82,6 +82,22 @@ type CopyOptions struct {
 	isPreserve       bool
 }
 
+// PutOptions holds options for object putting operation
+type PutOptions struct {
```

Comment:
Exporting this but none of the fields does not really make it usable.

Reply:
> Exporting this but none of the fields does not really make it usable.

Still this follows the existing style. I can send other PRs once this merged (this PR is not that easy to review in the first place)